doc/api/timers.markdown

```diff
@@ -7,15 +7,15 @@ this module in order to use them.
 
 ## clearImmediate(immediateObject)
 
-Stops an immediate from triggering.
+Stops an `immediateObject`, as created by [`setImmediate`][], from triggering.
 
 ## clearInterval(intervalObject)
 
-Stops an interval from triggering.
+Stops an `intervalObject`, as created by [`setInterval`][], from triggering.
```

Comment:
The actual objects are, for reference and naming, [`Timeout`](https://github.com/nodejs/node/blob/master/lib/timers.js#L442-L450) and [`Immediate`](https://github.com/nodejs/node/blob/master/lib/timers.js#L554)